unity/XRTProject/Assets/Scripts/Training/TrainingManager.cs

```diff
@@ -187,6 +187,25 @@ public void SwitchTask(Task task)
     {
         CurrentTask = task;
         OnCurrentTaskChanged?.Invoke();
+        if (task != null)
+        {
+            StartCoroutine(apiService.TextToSpeech(task.Description, (obj) =>
+            {
+                if (obj is BackendErrorResponse error)
+                {
+                    Debug.LogError($"Error generating speech {error.Status}: {error.Message}");
+                }
+                else if (obj is byte[] bytes)
+                {
+                    AudioClip audioClip = WavUtility.ToAudioClip(bytes);
+                    AudioManager.instance.AddClip(audioClip, "taskDescription");
+                    AudioManager.instance.Play("taskDescription");
```

Comment:
`AudioManager.instance...` works but should I be using `FindObjectOfType<AudioManager>()...`? Did not test with the latter